src/main/java/github/grace000/erversay/ServerConnection.java

```diff
@@ -0,0 +1,52 @@
+package github.grace000.erversay;
+
+import java.io.*;
+import java.net.*;
```

Comment:
Do these both need to be *?

Reply:
I can just import the individual files that I need.